plugin-server/src/main/services/http-server.ts

```diff
@@ -50,6 +44,24 @@ export function createHttpServer(hub: Hub, serverInstance: ServerInstance, serve
                 res.statusCode = 503
                 res.end(JSON.stringify(responseBody))
             }
+        } else if (req.url === '/_ready' && req.method === 'GET') {
+            // Check that, if the server should have a kafka queue,
+            // the Kafka consumer is ready to consume messages
+            if (!serverInstance.queue || serverInstance.queue.consumerReady) {
```

Comment:
Is there a point where we haven't set `serverInstance.queue` but will eventually? I'm thinking if there is a state where we'd mark it as ready but it isn't yet.

Reply:
Nope, you'll see in `pluginsServer.ts` that the queue is always set up before the server